### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by helenol**
Please let me know if there's any feedback on any of this - I tried to copy the naming as much from tf_conversions and eigen_conversions as possible, but use proper Google input/output style (pointers to output variables).

Unit tests in next PR, when I have some spare time between writing interfaces. :)

### CODE REVIEW

File: minkindr_conversions/include/minkindr_conversions/kindr_tf.h [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+#include <Eigen/Core>
+#include <Eigen/Geometry>
+#include <kindr/minimal/quat-transformation.h>
+#include <tf/transform_datatypes.h>
+
+namespace tf {
+
+// Convert a kindr::minimal::QuatTransformation to a 6 DoF geometry msgs pose.
+void poseKindrToTF(const kindr::minimal::QuatTransformation& kindr,
```
Comment (markusachtelik) [inline minkindr_conversions/include/minkindr_conversions/kindr_tf.h:12]: nit: TF --> Tf ? makes some functions below look less ugly.

File: minkindr_conversions/include/minkindr_conversions/kindr_tf.h [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+void quaternionTFToKindr(const tf::Quaternion& tf_type,
+                         kindr::minimal::RotationQuaternion* kindr);
+// Also the Eigen implementation version of this.
+void quaternionKindrToTF(const Eigen::Quaterniond& kindr,
+                         tf::Quaternion* tf_type);
+void quaternionTFToKindr(const tf::Quaternion& tf_type,
+                         Eigen::Quaterniond* kindr);
+
+// A wrapper for the relevant functions in eigen_conversions.
+void pointKindrToTF(const Eigen::Vector3d& kindr, tf::Vector3* tf_type);
```
Comment (markusachtelik) [inline minkindr_conversions/include/minkindr_conversions/kindr_tf.h]: wouldn't that be "vector" instead of "point" ?

**Comment by markusachtelik** [comment]
lgtm, thanks! re TF<-->Tf: let's do it the way like in eigen_conversions

**Comment by helenol** [comment]
Yeah I actually had Tf instead of TF originally - we're not sticking TOO solidly to their convention anyway, so I'm happy to change back to "Tf" if you decide. :) Camelcase with acronyms is always tough...

Good catch on point vs. vector, fixed and merging. :) Thanks a lot!

**Pull request merged**